package.json

```diff
@@ -61,8 +61,9 @@
     "currency-symbol-map": "5.0.1",
     "d3": "7.0.0",
     "dateformat": "3.0.3",
-    "dayjs": "1.10.4",
+    "dayjs": "1.10.7",
     "detect-port": "1.3.0",
+    "dotenv-webpack": "^7.0.3",
```

Comment:
Can we use the specific version for this?

Reply:
Thanks for catching!